l10n/en/products/vpn/resource-center.ftl

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/resource-center
+
+# HTML Page Title
+vpn-resource-center-title = Resource Center
+
+vpn-resource-center-whats-the-deal = What’s the deal with Virtual Private Networks?
+vpn-resource-center-discover-how-they = Discover how they work, why they’re important and what using a VPN can do for you.
+vpn-resource-center-mozilla-vpn = { -brand-name-mozilla-vpn }
+vpn-resource-center-start-protecting = Start protecting your identity today
+
+vpn-resource-center-obsessed-with = Obsessed with your privacy since 1998
+vpn-resource-center-get-mozilla-vpn = Get { -brand-name-mozilla-vpn }
+vpn-resource-center-was-this-article = Was this article helpful?
+
+# Variables:
+#   $amount (string) - percentage saved with chosen subscription plan e.g. '40'. With the von helper will out put 'Save $amount'
+# Added to the end of a translated string which would read 'Save 50% on Mozilla VPN'. Asterisk indicates a footnote for the following string
+vpn-resource-center-on-mozilla-vpn = { $amount } on { -brand-name-mozilla-vpn }*
```

Comment:
This string still feels a little off like we're mixing two separate translations, since `$amount` is essentially another localized string (`Save 50%`) as opposed to just a percentage. It _might_ be OK, but I wouldn't be too suprised if this caused issues for some locales. 

Maybe this is worth asking Peiying for advice on?